include/deal.II/base/types.h

```diff
@@ -41,6 +41,12 @@ namespace types
    */
   typedef unsigned int subdomain_id;
 
+  /**
+   * The type used for global indices of vertices.
+   */
+  typedef unsigned long long int global_vertex_index;
+#  define DEAL_II_VERTEX_INDEX_MPI_TYPE MPI_UNSIGNED_LONG_LONG
```

Comment:
I need global index for the vertices to couple deal.II with another library and they use ```unsigned long long int```. I can use global_dof_index if you prefer.